modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/java/com/liferay/headless/builder/test/HeadlessBuilderTest.java

```diff
@@ -255,6 +274,9 @@ private void _withHeadlessBuilderApplication(
 		}
 	}
 
+	private static final String _OBJECT_FIELD_NAME_1 =
```

Comment:
Maybe better to remove the 1 from `_OBJECT_FIELD_NAME_1` as there is only one name?

Reply:
Was already changed in other commits of the PR